packages/react/src/TreeView/TreeView.tsx

```diff
@@ -187,10 +193,14 @@ const UlBox = styled.ul<SxProp>`
   .PRIVATE_TreeView-item-content {
     grid-area: content;
     display: flex;
-    align-items: center;
+    align-items: flex-start;
```

Comment:
I am not really sure if we need the flex-start here in the content 🤔 Same as line-height just above.